SECURITY.md

```diff
@@ -0,0 +1,71 @@
+# Security Policy
+
+## No guarantees
+
+Support is provided on a best-effort bases only.
+No binding guarantees can be provided.
+
+## Security premises
+
+Rand provides the trait `rand_core::CryptoRng` aka `rand::CryptoRng` as a marker
+trait. Generators implementating `RngCore` *and* `CryptoRng`, and given the
+additional constraints that:
+
+-   Instances of seedable RNGs (those implementing `SeedableRng`) are
+    constructed with cryptographically secure seed values
+-   The state (memory) of the RNG and its seed value are not be exposed
+
+are expected to provide the following:
+
+-   Given all prior output from the RNG, for each bit of non-error output from
+    an `RngCore` method, there exists no known computationally-feasible
+    algorithm to predict the bit with probability significantly greater than 50%
+-   There are strong expectations that no such algorithm exists
```

Comment:
I'd personally word this something like:

> An attacker can gain no advantage over chance (50%) in predicting the RNG output, even with full knowledge of all prior outputs.

That's mostly to say that the latter bit is a red herring if the attacker doesn't need it to predict future outputs (because the attacker has a covert channel to the internal state, or can manipulate the internal state)

Reply:
Done